src/App.tsx

```diff
@@ -43,20 +53,30 @@ function App() {
           icon={faCoffee}
         />
       </div>
-      <Title size="h1" text="Dit is een titel h1" />
-      <Title size="h2" text="Dit is een titel h2" />
-      <Title size="h3" text="Dit is een titel h3" />
-      <Title size="h4" text="Dit is een titel h4" />
-      <Title size="h5" text="Dit is een titel h5" />
+      <Title size={TitleSizeEnum.H1} text="Dit is een titel h1" />
+      <Title size={TitleSizeEnum.H2} text="Dit is een titel h2" />
+      <Title size={TitleSizeEnum.H3} text="Dit is een titel h3" />
+      <Title size={TitleSizeEnum.H4} text="Dit is een titel h4" />
+      <Title size={TitleSizeEnum.H5} text="Dit is een titel h5" />
 
-      <Text variant="small" text="Dit is smalle tekst" />
-      <Text variant="normal" text="Dit is smalle tekst" />
-      <Text variant="quote" text="Dit is smalle tekst" />
-      <Text variant="lead" text="Dit is smalle tekst" />
+      <Text variant={TextVariantEnum.SMALL} text="Dit is smalle tekst" />
+      <Text variant={TextVariantEnum.NORMAL} text="Dit is smalle tekst" />
+      <Text variant={TextVariantEnum.QUOTE} text="Dit is smalle tekst" />
+      <Text variant={TextVariantEnum.LEAD} text="Dit is smalle tekst" />
 
       <HyperLink href="https://www.google.com" className="nav-animation" text="Navlink" />
       <br />
       <HyperLink href="https://www.google.com" className="tw-underline" text="Normale link" />
+      <div className="tw-mt-10">
+        <InputField
+          onChange={change}
```

Comment:
Can you change `onChange={change}` to `onChange={(e) => setVal(e.target.value)}`
and remove the change function